lib/inc/pxp-agent/configuration.hpp

```diff
@@ -35,7 +35,7 @@ struct EntryBase {
     // Config option name
     // HERE: must match one of the flag names and config file option
     std::string name;
-    // CLI option aliases (e.g. --server --serv -s)
+    // CLI option aliases (e.g. --broker --bro -b)
```

Comment:
I think we should get rid of the alias. --broker isn't very long and nothing else is shortened.